keyboards/splitreus62/readme.md

```diff
@@ -0,0 +1,17 @@
+# splitreus62
+
+- https://github.com/Na-Cly/splitreus62
+- [Atreus](https://github.com/technomancy/atreus) Phil Hagelberg
+- [Atreus62](https://github.com/profet23/atreus62) Profet23
+
+A Split version of the Atreus62. You can copy keymaps from Atreus62 and use them with this keyboard.
```

Comment:
I think this is a bit cleaner, but it retains all the information. Your link is under Hardware Availability, so it doesn't need to be in the preamble.

```suggestion
A split version of the [Atreus62](https://github.com/profet23/atreus62) by Profet23, which itself is based on Phil Hagelberg's [Atreus](https://github.com/technomancy/atreus). You can copy keymaps from Atreus62 and use them with this keyboard.
```